dolphin/processor/config.py

```diff
@@ -289,10 +289,21 @@ def get_kwargs_likelihood(self):
                     prior_param.extend(i)
                     kwargs_likelihood['prior_ps'].append(prior_param)
 
-        if 'lens_option' in self.settings and \
-            'constrain_position_angle_from_lens_light' \
-                in self.settings['lens_option']:
-            kwargs_likelihood['custom_logL_addition'] = \
+        use_custom_logL_addition = False
+
+        if 'lens_option' in self.settings:
+            if 'constrain_position_angle_from_lens_light'\
+                    in self.settings['lens_option']:
+                use_custom_logL_addition = True
+            if 'max_qm/qi_ratio' in self.settings['lens_option']:
+                use_custom_logL_addition = True
+
+        if 'source_light_option' in self.settings:
+            if 'jeffreys_beta' in self.settings['source_light_option']:
```

Comment:
This may not be very self-explanatory to everyone. Please use `'shapelet_scale_logarithmic_prior'` instead.